File: data/python/CALCULATE_VOLUME_DODECAHEDRON_MAIN.py
if __name__ == '__main__':
    param = [
    (56,),
    (73,),
    (22,),
    (10,),
    (84,),
    (20,),
    (51,),
    (91,),
    (10,),
    (83,)
        ]
    n_success = 0
    print("class,mutant,test_index,result\n")
    for i, parameters_set in enumerate(param):
        try:
            print("CALCULATE_VOLUME_DODECAHEDRON," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("CALCULATE_VOLUME_DODECAHEDRON," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/CALCULATE_VOLUME_DODECAHEDRON_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(56);
    param0.add(73);
    param0.add(22);
    param0.add(10);
    param0.add(84);
    param0.add(20);
    param0.add(51);
    param0.add(91);
    param0.add(10);
    param0.add(83);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("CALCULATE_VOLUME_DODECAHEDRON," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("CALCULATE_VOLUME_DODECAHEDRON," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
